src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/EditAndSyncFeatures/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "Edit and sync features",
+  "SampleName": "EditAndSyncFeatures",
+  "Description": "This sample demonstrates how to synchronize offline edits with a feature service.",
+  "Instructions": "1. Pan and zoom to the area you would like to download features for, ensuring that all features are within the rectangle.\n2. Tap the 'generate' button. This will start the process of generating the offline geodatabase.\n3. Tap on a feature within the area of the generated geodatabase. Then tap on the screen (anywhere within the range of the local geodatabase) to move the feature to that location.\n4. Tap the 'Sync Geodatabase' button to synchronize the changes back to the feature service.\n\n Note that the basemap for this sample is downloaded from ArcGIS Online automatically.",
+  "Type": 0,
+  "RequiresOnlineConnection": true,
+  "RequiresOfflineData": true,
+  "RequiresLocalServer": false,
+  "DataItemIds": [ "3f1bbf0ec70b409a975f5c91f363fe7d" ],
+  "Image": "EditAndSyncFeatures.jpg",
+  "Link": "",
+  "TypeLink": []
```

Comment:
You should add as many Esri.ArcGISRuntime.xml Types to the TypeLink section as you can so that the sample can be picked up in the Library Reference Documentation. Ex: "M:Esri.ArcGISRuntime.Tasks.Offline.GeodatabaseSyncTask.CreateAsync(System.Uri,Esri.ArcGISRuntime.Security.Credential)","M:Esri.ArcGISRuntime.Tasks.Offline.GeodatabaseSyncTask.CreateDefaultGenerateGeodatabaseParametersAsync(Esri.ArcGISRuntime.Geometry.Envelope)","M:Esri.ArcGISRuntime.Tasks.Offline.GeodatabaseSyncTask.GenerateGeodatabase(Esri.ArcGISRuntime.Tasks.Offline.GenerateGeodatabaseParameters,System.String)","T:Esri.ArcGISRuntime.Tasks.Offline.GenerateGeodatabaseJob","T:Esri.ArcGISRuntime.Tasks.Offline.SyncGeodatabaseParameters","T:Esri.ArcGISRuntime.Tasks.Offline.SyncDirection","T:Esri.ArcGISRuntime.Tasks.Offline.SyncLayerOption","T:Esri.ArcGISRuntime.Tasks.Offline.SyncGeodatabaseJob","M:Esri.ArcGISRuntime.Tasks.Offline.GeodatabaseSyncTask.SyncGeodatabase(Esri.ArcGISRuntime.Tasks.Offline.SyncGeodatabaseParameters,Esri.ArcGISRuntime.Data.Geodatabase)"